frontend/src/common/DisabledElementTooltip.spec.tsx

```diff
@@ -10,7 +10,7 @@ describe('DisabledElementTooltip', () => {
   it('renders an DisabledElementTooltip component when a button is hovered over', async () => {
     render(
       <DisabledElementTooltip conditions={[testMessage]}>
-        <Button data-testid='trigger'>Button</Button>
+        <Button data-test='trigger'>Button</Button>
```

Comment:
Docs for testing-library/react suggest that we should use `data-testid`, was this not working?

Reply:
We've changed this in our config. I didn't want us to have different test ids for cypress and unit tests. So have configured it to use data-test for both. See testUtils.ts for the config change.